README.md

```diff
@@ -1,2 +1,135 @@
+- The orientation estimation Roll, Pitch and Yaw.
+- The accelerometer measuremtents x, y, z components.
+- The gyroscope measuremtents x, y, z components.
+- The magnetometer measuremtents x, y, z components.
+
+## [Dependencies](#top)
+
+Server dependencies:
+- [YARP](https://github.com/robotology/yarp): Middleware for handling the communication with the robot.
+- [node.js](https://nodejs.org/en/): Asynchronous event-driven JavaScript runtime, designed to build scalable network applications.
+- [npm](https://www.npmjs.com/): Node package manager.
+- [YarpJS](https://github.com/robotology/yarp.js): Yarp Javascript bindings
+- [Open MCT](https://github.com/nasa/openmct): Open source visualization framework by NASA.
+
+Client dependencies:
+- Browser: [Google Chrome](https://www.google.com/chrome), [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/products), Apple Safari, etc.
+
+
+## [Server Installation](#top)
+
+The following instructions assume you are installing the software as a non-root user. Make sure that you have [Git](https://git-scm.com) and the . The installation and run has been tested on MacOS Catalina 10.15.7.
+
+1. Install **YARP**: it is recommended to install the binaries using the [Conda package manager](https://anaconda.org/) and installing the YARP package from the [Robotology Conda channel](https://anaconda.org/robotology).
+    - If you are not already using the `conda` package manager, install the `conda` miniforge distribution following https://github.com/robotology/robotology-superbuild/blob/master/doc/install-miniforge.md#linux. Remember to restart your shell session or run `source ~/.bashrc` (`~/.bash_profile` on MacOS) for the `conda init` command to take effect.
+    - Install Mamba, create a new environment and install the YARP package:
+    ```
+    conda install mamba
+    conda create -n robotologyenv
+    conda activate robotologyenv
+    mamba install -c robotology yarp
+    ```
+    To read more about installing `robotology` package binaries refer to https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-forge.md#binary-installation.
+2. Install **node.js** and **npm**: you can follow the official guides in the respective web pages but we recommend to install them through [NVM](). Currently, the latest **node.js** LTS version compatible with **YarpJS** `master` commit [be28630](https://github.com/robotology/yarp.js/commit/be2863022713ded2fa48909404b43e98b09eeda2) is the LTS: Argon release **node.js v4.2.2** (refer to https://github.com/robotology/yarp.js/issues/19). Meanwhile, the latest **node.js** LTS version compatible with **Open MCT** is latest LTS:Fermium version **node.js v14.17.0**. For this reason we install both releases.
```

Comment:
The NVM link is missing. Furthermore, the structure of the sentence is a bit misleading: it starts with the "follow the official guides in the respective web pages", but that not the option we recommend. Perhaps we can write: 
```suggestion
2. Install **node.js** and **npm**: we recommende to install them via  [NVM](). Currently, the latest **node.js** LTS version compatible with **YarpJS** `master` commit [be28630](https://github.com/robotology/yarp.js/commit/be2863022713ded2fa48909404b43e98b09eeda2) is the LTS: Argon release **node.js v4.2.2** (refer to https://github.com/robotology/yarp.js/issues/19). Meanwhile, the latest **node.js** LTS version compatible with **Open MCT** is latest LTS:Fermium version **node.js v14.17.0**. For this reason we install both releases.
```

Reply:
Ok, I guess that "you can follow the official ..." can be assumed by default, so leaving just "we recommend" is more clear. There's a typo at "recommende".

Reply:
Fixed in https://github.com/dic-iit/yarp-openmct/pull/2/commits/df23358e9bad17f9b87871dd76832d2948df6f72.